plugins/amazonq/codewhisperer/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeFileScanTest.kt

```diff
@@ -54,17 +57,27 @@ class CodeWhispererCodeFileScanTest : CodeWhispererCodeScanTestBase(PythonCodeIn
     private lateinit var psifile2: PsiFile
     private lateinit var psifile3: PsiFile
     private lateinit var psifile4: PsiFile
+    private lateinit var psifile5: PsiFile
```

Comment:
approved but nit: can we change names of these files that are being used for performance tests.